es-app/src/SystemData.h

```diff
@@ -8,23 +8,31 @@
 #include "PlatformId.h"
 #include "ThemeData.h"
 #include "FileFilterIndex.h"
+#include "CollectionSystemManager.h"
+
+struct SystemEnvironmentData
```

Comment:
...would you believe me if I said I did that to mess with you? :P 

Thanks for spotting that, it's obviously an oversight from my end! Glad you caught that now, otherwise in the future someone would certainly look at that and be utterly confused.

It should be a struct.